web/packages/design/src/theme/theme.ts

```diff
@@ -31,27 +31,21 @@ const space = [0, 4, 8, 16, 24, 32, 40, 48, 56, 64, 72, 80];
 const contrastThreshold = 3;
 
 const colors = {
-  accent: '#651FFF',
-  progressBarColor: '#00BFA5',
-
-  dark: '#000',
+  levels: {
+    sunken: '#0C143D',
+    sunkenSecondary: '#111B48',
 
-  light: '#FFFFFF',
+    surface: '#222C59',
+    surfaceSecondary: '#1C254D',
 
-  primary: {
-    darker: '#0C143D',
-    main: '#1C254D',
-    light: '#222C59',
-    lighter: '#2C3A73',
-    dark: '#111B48',
-    contrastText: '#FFFFFF',
+    elevated: '#2C3A73',
+    popout: '#3E4B7E',
   },
 
-  secondary: {
+  brand: {
```

Comment:
> i'm seeing brand used mostly to mean as a `background` color. but i think i also saw it used sparingly as a text color, input box hover? or border color.

The `brand` colours are the bright purple accent colours that are associated with Teleport (eg. our logo), I think it makes sense to keep them defined under `brand` rather than creating subcategories for specific components, since these are usually used minor accent colours and I don't think we will ever have components using different brand-themed purple accent colours.

I agree with @ravicious in that we should only need to break down our theme into subcategories ie `colors.buttons` when the components have different kinds with each kind having a unique set of colours. In the case of `buttons`, each button `kind` has varying background colours, border colour, colours for hover states etc., this makes it necessary to subcategorize them as opposed to having a large amount of colours defined directly in `theme.colors` such as `theme.colors.buttonPrimaryDefault`, `theme.colors.buttonPrimaryHover`, `theme.colors.buttonBorderBorderHover` etc.

So far, `buttons` has been the only component type where I have found this to be necessary. However with the new theme, there likely will be new unique designs for things like inputs and labels where it would make sense to do as you suggested and break them down into their own subgroupings. The `theme` in this PR is not final and there will likely be more cases introduced by the new themes that it needs to handle.